src/mscorlib/src/System/IO/UnmanagedMemoryAccessor.cs

```diff
@@ -1019,73 +998,6 @@ public class UnmanagedMemoryAccessor : IDisposable {
             }
         }
 
-        // Writes the struct pointed to by ref value into unmanaged memory.  Note that this method
-        // is most performant when used with medium to large sized structs (larger than 8 bytes 
-        // though this is number is JIT and architecture dependent).   As such, it is best to use 
-        // the WriteX methods for small standard types such as ints, longs, bools, etc.
-
-        [System.Security.SecurityCritical]  // auto-generated_required
-        public void Write<T>(Int64 position, ref T structure) where T : struct {
-            if (position < 0) {
```

Comment:
(You can add it to model.xml while you are on it to safe somebody time later...)